packages/foreman/foreman/foreman.spec

```diff
@@ -698,14 +698,14 @@ plugins required for Foreman to work.
 #replace shebangs and binaries in scripts for SCL
 %if 0%{?scl:1}
   # shebangs
-  for f in bin/* script/performance/profiler script/performance/benchmarker script/dynflowd ; do
+  for f in bin/* script/performance/profiler script/performance/benchmarker script/dynflowd; do
     sed -ri '1sX(/usr/bin/ruby|/usr/bin/env ruby)X%{scl_ruby_bin}X' $f
   done
   # script content
   sed -ri 'sX/usr/bin/rakeX%{scl_rake}X' extras/dbmigrate script/foreman-rake
-
-  sed -i '/^ExecStart/ s|/usr/bin/sidekiq \(.\+\)$|/usr/bin/scl enable tfm "sidekiq \1"|' extras/systemd/%{dynflow_sidekiq_service_name}.service
 %endif
+# sidekiq service SELinux helper path update
+sed -i '/^ExecStart/ s|/usr/bin/sidekiq \(.\+\)$|%{_libexecdir}/%{name}/sidekiq-selinux \1|' extras/systemd/%{dynflow_sidekiq_service_name}.service
```

Comment:
Debian also patches this:

https://github.com/theforeman/foreman-packaging/blob/21e0bef50d59a25e8c4b7c48934c3ffb0da770f9/debian/bionic/foreman/rules#L28

I'm wondering if we should make the change in `foreman.git` (https://github.com/theforeman/foreman/blob/9b487e87b29e9a2ec5dfaa91b25ff2c9e73cc566/extras/systemd/dynflow-sidekiq%40.service#L18)

@evgeni thoughts?

Reply:
Whatever, but keep in mind I am trying to put this into 2.1, this is a critical feature for ReX. If you insist on this, I'd ask you to shorten the review loop so I can get this through.

Don't have a strong opinion on this other than `/usr/bin/sidekiq` looks like a reasonable default :)

Reply:
there is no reasonable catch all default that would work for debian and centos, so having a generic path and fix that up seems OK to me.